src/screens/Exchange/ExchangeConfirm/ExchangeConfirm.js

```diff
@@ -150,7 +150,7 @@ const ExchangeConfirmScreen = () => {
 
           {highFee && (
             <WarningBlock
-              text={t('transactions.highGasFee.warningLabel')}
+              text={t('transactions.highGasFee.ethereumNetworkFees')}
```

Comment:
This was set as `warningLabel` as the message is blockchain agnostic, whereas `ethereumNetworkFees` will display a message about the Ethereum network when the high gas fee may be on Polygon, for example. See: https://github.com/pillarwallet/pillarwallet/pull/3042

Reply:
Okay, I'll revert it.